jscomp/bsb/bsb_file_groups.mli

```diff
@@ -17,58 +17,59 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
- * 
+ *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. *)
 
 
- type public = 
+ type public =
   | Export_none
-  | Export_all 
-  | Export_set of Set_string.t 
-  
+  | Export_all
+  | Export_set of Set_string.t
 
-type build_generator = 
+
+type build_generator =
   { input : string list ;
     output : string list;
-    command : string}  
+    command : string}
 
 
-type  file_group = 
+type  file_group =
   { dir : string ;
-    sources : Bsb_db.map; 
+    subdirs : string list;
```

Comment:
real change in this file is adding this field. The rest is trailing whitespace getting removed......